pkg/common/utils/gang_utils_test.go

```diff
@@ -289,3 +289,45 @@ func TestGetTaskGroupFromAnnotation(t *testing.T) {
 	assert.Equal(t, taskGroups2[0].MinResource["cpu"], resource.MustParse("2"))
 	assert.Equal(t, taskGroups2[0].MinResource["memory"], resource.MustParse("1Gi"))
 }
+
+func TestGetPlaceholderTimeoutParam(t *testing.T) {
```

Comment:
NIT: can we optimize this to be table-driven style, you can find some doc here: https://gobyexample.com/testing.

Reply:
Understood.
I will optimize this with table-driven style.